src/server/api/routes/possibleanswer.router.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const possibleanswerController = require('../controllers/possibleanswer.controller');
+
+/**
+ * @swagger
+ * /answer_choices/{ID}:
+ *  get:
+ *    summary: Get answer_choices by ID
+ *    description:
+ *      Will return single answer_text with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the answer_choices to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/:id', (req, res, next) => {
```

Comment:
Possibleanswer is a little vague in its description, imo.

Reply:
This is what i could think of because it is Possibleanswer for a question. What do you suggest??

Reply:
Ok. What about GameQuestionAnswers then?

Reply:
That's a good name but this endpoint will provide 4 options as an answer so GameQuestionAnswerOptions ??